packages/create-vite/src/index.ts

```diff
@@ -249,6 +249,25 @@ const FRAMEWORKS: Framework[] = [
       },
     ],
   },
+  {
+    name: 'angular',
+    display: 'Angular',
+    color: red,
+    variants: [
+      {
+        name: 'custom-angular',
+        display: 'Angular ↗',
+        color: red,
+        customCommand: 'npx -p @angular/cli@latest ng new TARGET_DIR',
```

Comment:
@gioboa Could this be `npm exec @angular/cli@latest new TARGET_DIR` instead? `create-vite` has special handling to execute the CLI with yarn/pnpm/bun dynamically if `npm exec` here is used, depending on the package manager that invoked `create-vite` in the first place.

Reply:
Yep, it's working. 
I created the PR https://github.com/vitejs/vite/pull/18425
Thanks